### PR TITLE
feat: add arbitrum

### PR DESCRIPTION
**Pull request by kaladinlight**

### CODE REVIEW

File: packages/hdwallet-tallyho/src/tallyho.ts [hunk 1]
```diff
@@ -92,6 +92,7 @@ export class TallyHoHDWallet implements core.HDWallet, core.ETHWallet {
   readonly _supportsOptimism = false;
   readonly _supportsPolygon = true;
   readonly _supportsGnosis = false;
+  readonly _supportsArbitrum = true;
```
Comment (gomesalexandre) [inline packages/hdwallet-tallyho/src/tallyho.ts:95]: FWIW, doesn't seem live yet https://taho.upvoty.com/b/taho-roadmap/arbitrum-support/

File: packages/hdwallet-walletconnect/src/walletconnect.ts [hunk 2]
```diff
@@ -122,6 +122,7 @@ export class WalletConnectHDWallet implements core.HDWallet, core.ETHWallet {
   readonly _supportsBSC = false;
   readonly _supportsPolygon = false;
   readonly _supportsGnosis = false;
+  readonly _supportsArbitrum = false;
```
Comment (gomesalexandre) [inline packages/hdwallet-walletconnect/src/walletconnect.ts:125]: Was there a rationale for hdwallet-walletconnect declaring all EVM chains except Ethereum mainnet as unsupported?

**Review by gomesalexandre — APPROVED** [review]
Two comments WRT feature capabilities but conceptual stamp on this

File: packages/hdwallet-tallyho/src/tallyho.ts (hunk 1 above)
Comment (kaladinlight) [reply packages/hdwallet-tallyho/src/tallyho.ts:95]: It's marked live and showing in the wallet:
image
image

File: packages/hdwallet-walletconnect/src/walletconnect.ts (hunk 2 above)
Comment (gomesalexandre) [reply packages/hdwallet-walletconnect/src/walletconnect.ts:125]: Consistency sounds sane to me! We can follow-up with an attempt to enable all chains as a follow-up

**Pull request merged**